python/GafferArnoldUI/ArnoldOptionsUI.py

```diff
@@ -111,6 +111,12 @@ def __subdivisionSummary( plug ) :
 	info = []
 	if plug["maxSubdivisions"]["enabled"].getValue():
 		info.append( "Max Subdivisions  %d" % plug["maxSubdivisions"]["value"].getValue() )
+	if plug["subdivDicingCamera"]["enabled"].getValue():
+		info.append( "Dicing Camera %s" % plug["subdivDicingCamera"]["value"].getValue() )
+	if plug["subdivFrustumCulling"]["enabled"].getValue():
+		info.append( "Frustum Culling %f" % plug["subdivFrustumCulling"]["value"].getValue() )
+	if plug["subdivFrustumPadding"]["enabled"].getValue():
+		info.append( "Frustum Padding %f" % plug["subdivFrustumPadding"]["value"].getValue() )
```

Comment:
Use `GafferUI.NumericWidget.valueToString()` for better formatting of float values.